### DISCUSSION TITLE
Real Time Code Plagiarism and Contestant Removal

### DISCUSSION THREAD

**Discussion by kittusinghkumar** [scaleracademy/open-source-projects#108] [Ideas]
---
Real Time Code Plagiarism and Contestant Removal
---
A Real time  Code Plagiarism detector which detects the contestants who copies code and remove them as soon as they submit their submission on the online coding platforms.

**Comment by DK-Khambhadiya** [comment]
It's very great idea! we can make use of many libraries such as puppeteer, we can use web scrapper for plagiarism checks.
